string local_leader
---
bool result
---
bool status

